docs/usage_guide/implementation.rst

```diff
@@ -0,0 +1,77 @@
+Implementation
+==============
+
+Extending a page to be personalisable
+-------------------------------------
+Wagxperience offers a ``PersonalisablePage`` base class to extend from.
+This is a standard ``Page`` class with personalisation options added.
+
+
+Creating a new personalisable page
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Import and extend the ``personalisation.models.PersonalisablePage`` class to
+create a personalisable page.
+
+A very simple example for a personalisable homepage:
+
+.. code-block:: python
+
+    from wagtail.wagtailcore.models import Page
+    from wagtail_personalisation.models import PersonalisablePageMixin
+
+    class HomePage(PersonalisablePageMixin, Page):
+        pass
+
+All you need is the ``PersonalisablePageMixin`` mixin and a Wagtail ``Page``
+class of your liking.
+
+
+Migrating an existing page to be personalisable
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Coming soon...
```

Comment:
in plaats van ``coming soon``, uit de documentatie laten en een issue aanmaken om dit op te pakken?